examples/templateservicebroker/apiserver-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: TemplateServiceBrokerConfig
+apiVersion: config.templateservicebroker.openshift.io/v1
+templateNamespaces:
+- openshift
```

Comment:
> But breaking that template into sub-files feels like a step too far to me.

That's definitely necessary since roles and rolebindings aren't `applied`, they are `reconciled`.



> I guess I'd just prefer to see/hear that the ansible installer really needs this as a standalone file before introducing it.

@sdodson you requested this separately so that you could have a clear division of responsibility: this is your config file and you modify it using our yaml tools versus this is your template and you set parameters.